docs/user/FlowVariables.md

```diff
@@ -128,8 +128,8 @@ Note:
 | `RTLMP_FLOW`          | Enable the Hierarchical RTLMP flow. By default it is disabled.                                                                                                                   |
 | `MACRO_PLACEMENT`     | Specifies the path of a file on how to place certain macros manually using read_macro_placement.                                                                                 |
 | `MACRO_PLACEMENT_TCL` | Specifies the path of a TCL file on how to place certain macros manually.                                                                                                        |
-| `MACRO_PLACE_HALO`    | horizontal/vertical halo around macros (microns).                                                                                                                                |
-| `MACRO_PLACE_CHANNEL` | horizontal/vertical channel width between macros (microns).                                                                                                                      |
+| `MACRO_PLACE_HALO`    | horizontal/vertical halo around macros (microns). Used by automatic macro placement, mpl and mpl2.                                                                               |

```

Comment:
It would be clearer to refer to the RTLMP_FLOW control rather than the code modules as this is flow documentation.